spec.md

```diff
@@ -559,10 +559,53 @@ The tags MUST be in lexical order.
 
 When using the `last` query parameter, the `n` parameter is OPTIONAL.
 
+Registries MAY support an enhanced tag listing experience when clients pass an `Accept` header value of `application/vnd.oci.image.index.v1+json`.
+If a registry supports this mode, they MUST set the `Content-Type: application/vnd.oci.image.index.v1+json` header.
+Registries supporting the enhanced tag list MUST support these additional query parameters:
+* `sort_by` - allowed values are `created_at`, `name`
+* `sort_order` - allowed values are `asc` and `desc`
+
+The response MUST be a json body in the following format:
+```json
+{
+  "schemaVersion": 2,
+  "mediaType": "application/vnd.oci.image.index.v1+json",
+  "manifests": [
+    {
+      "mediaType": "application/vnd.oci.image.manifest.v1+json",
+      "size": 1234,
+      "digest": "sha256:a1a1a1...",
+      "annotations": {
+        "org.opencontainers.image.ref.created": "2022-01-01T14:42:55Z",
+        "org.opencontainers.image.ref.name": "<tag1>"
+      }
+    },
+    {
+      "mediaType": "application/vnd.oci.image.manifest.v1+json",
+      "size": 1234,
+      "digest": "sha256:a2a2a2...",
+      "annotations": {
+        "org.opencontainers.image.ref.created": "2022-01-01T07:21:33Z",
+        "org.opencontainers.image.ref.name": "<tag2>"
+      }
+    }
+  ]
+}
+```
+The following annotations MUST be included:
+* `org.opencontainers.image.ref.created` - Date and time the reference was created (date-time string as defined by RFC 3339). If unknown, default to epoch.
```

Comment:
A resource usually has two timestamps: one for creation and another for modification. For example, tag `v1` is created on `2022-01-01T07:21:33Z` and later re-tagged on `2022-02-03T08:21:21Z`. Do you want to introduce a new annotation for the modification / update time?

Reply:
I'd say this could be an optional value added by registries, but we can specify an annotation for it. Some registries may not have this information, and it might be of little value (it wouldn't give a history) on it's own.